cypress/page-objects/Signup.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress" />
```

Comment:
In general, page object are clases. 
<img width="576" alt="Captura de Pantalla 2021-06-22 a la(s) 17 18 37" src="https://user-images.githubusercontent.com/11221560/122993715-01889380-d37e-11eb-8e3d-932adbe6e9b8.png">